### PR TITLE
FE: collab with Robert, added title and description + styled dashboar…

### PR DESCRIPTION
**Pull request by mpcavellier**
…d on club director dashboard

### CODE REVIEW

**Review by fidan93 — APPROVED** [review]
Looks good. As I understood that commented out code on line 40 is still necessary ? Just commented out for now?

File: src/components/pages/ClubsPages/anderson.css [hunk 1]
```diff
@@ -1,8 +1,41 @@
...
+.Plot{
+    width: 100%;
+    height:100vh;
+    font: 200px;
+}
+
+.graph-holder{
+    width:100%;
+    height:115vh;
+
```
Comment (Shawn2776) [inline src/components/pages/ClubsPages/anderson.css:14]: Maybe get rid of some extra space here and below

**Review by Shawn2776 — APPROVED** [review]
Good job. Changes approved.

**Comment by mpcavellier** [comment]
Co-authored-by: name robertmasters@hotmail.com

**Pull request merged**